apps/mobile/src/navigations/root/root.tsx

```diff
@@ -29,6 +30,10 @@ const Root: React.FC = () => {
 				/>
 			) : (
 				<NativeStack.Group>
+					<NativeStack.Screen
+						component={Welcome}
+						name={RootScreenName.WELCOME}
+					/>
```

Comment:
I believe we should have this screen when the user is logged it

Reply:
Regarding the design this screen should be after Onboarding questions. We'll have a separate ticket for screens order later.

Reply:
Didn't notice that it is in the Auth group. Yes, please move it above